fstab.yaml

```diff
@@ -1,2 +1,2 @@
 mountpoints:
-  /: https://drive.google.com/drive/folders/1kTGAtW7B6uG1JKhcNgvP7q1iNY-XP3eR 
+  /: https://drive.google.com/drive/folders/14DtFfIeyOTso7-Sc5JqpgVpH496yBJcI
```

Comment:
Are you using a copy of the gdrive folder to test your changes?

Reply:
yes, i need to change that before merging. thanks for spotting it out.